sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -848,14 +856,37 @@ private boolean keepColumn(ImportColumn column, String prefix, List<String> colu
 		return true;
 	}
 
-	public boolean executeDefaultInvokings(PropertyDescriptor pd, Object element, String entry, String[] entryHeaderPath)
-		throws InvocationTargetException, IllegalAccessException, ParseException, ImportErrorException {
+	public boolean executeDefaultInvokings(PropertyDescriptor pd, Object element, String entry, String[] entryHeaderPath, boolean allowForeignRegions)
+		throws InvocationTargetException, IllegalAccessException, ParseException, ImportErrorException, EnumService.InvalidEnumCaptionException {
 		Class<?> propertyType = pd.getPropertyType();
 
 		if (propertyType.isEnum()) {
-			pd.getWriteMethod().invoke(element, Enum.valueOf((Class<? extends Enum>) propertyType, entry.toUpperCase()));
+			Enum enumValue = null;
+			Class<Enum> enumType = (Class<Enum>) propertyType;
+			try {
+				enumValue = Enum.valueOf(enumType, entry.toUpperCase());
+			} catch (IllegalArgumentException e) {
+				// ignore
+			}
+
+			if (enumValue == null) {
+				enumValue = enumService.getEnumByCaption(enumType, entry);
+			}
+
+			pd.getWriteMethod().invoke(element, enumValue);
 			return true;
 		}
+		if (propertyType.getSuperclass() != null && propertyType.getSuperclass() == CustomizableEnum.class) {
+			try {
+				Object test = propertyType.newInstance();
+				((CustomizableEnum) test).setValue(entry);
```

Comment:
Replaced the name. There's currently no way to test this because that would require at least one customizable enum value to be persisted in the test database, and we don't have backend logic to create these values yet.